src/netstandard/ref/System.Runtime.CompilerServices.cs

```diff
@@ -529,6 +529,7 @@ public static partial class RuntimeHelpers
         public static void ExecuteCodeWithGuaranteedCleanup(System.Runtime.CompilerServices.RuntimeHelpers.TryCode code, System.Runtime.CompilerServices.RuntimeHelpers.CleanupCode backoutCode, object userData) { }
         public static int GetHashCode(object o) { throw null; }
         public static object GetObjectValue(object obj) { throw null; }
+        public static T[] GetSubArray<T>(T[] array, System.Range range) { throw null; }
```

Comment:
Not to block this PR, but can you remind me why this is being added to RuntimeHelpers rather than to Array?

Reply:
I don’t think we ever considered Array. We put it on runtime helpers because we don’t expect anyone but the compiler yo call this (the compilers will synthesize  the indexer in the arry symbol and lower the invocation to calling this method).